docs/sources/clients/promtail/scraping.md

```diff
@@ -109,7 +110,8 @@ here for reference. The `max_age` field ensures that no older entry than the
 time specified will be sent to Loki; this circumvents "entry too old" errors.
 The `path` field tells Promtail where to read journal entries from. The labels
 map defines a constant list of labels to add to every journal entry that Promtail
-reads.
+reads. The `matches` field adds journal filters (the character `+` is not supported,
+only logical `AND` filters are added).
```

Comment:
If we make `matches` a list, we can probably change this to:
```suggestion
reads. Multiple `matches` can be defined as a list of journal filters, which must _all_ match in order for the entry to be considered.
```

Reply:
I think we need to refrase it again... journald is a bit tricky, [sd_journal_add_match](https://github.com/systemd/systemd/blob/main/src/libsystemd/sd-journal/sd-journal.c#L290) change an AND for an OR when the filter is the same.

It's also documented in the man page:
> If multiple matches are specified
       matching different fields, the log entries are filtered by both,
       i.e. the resulting output will show only entries matching all the
       specified matches of this kind. If two matches apply to the same
       field, then they are automatically matched as alternatives, i.e.
       the resulting output will show entries matching any of the
       specified matches for the same field.